README.md

```diff
@@ -80,6 +84,30 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
+## Dasherized Fields
+
+JSONAPI now recommends the use of dashes (`-`) in place of underscore (`_`) as a
```

Comment:
I'd be curious as to @doomspork or @jeregrine opinion what the library `case` default should be and if we should take into consideration camelCase recommendation now.  Or should we stick with general Elixir conventions for underscore and leave as it, providing configuration on top to convert to `dash` or `camelCase` based on what the consumer wants?

Reply:
As someone using this lib I'd prefer to opt into whatever I want, even on a request level. Imagine an API like Stripe's where you tell the API what version of it you're expecting. In such a case I'd want to either dasherize for old clients, or camelize for modern ones. FWIW as a default I'd love to see at least dashes given that this is the v1.0 recommendation.